TESTS/mbedmicro-rtos-mbed/malloc/main.cpp

```diff
@@ -122,7 +122,7 @@ void test_alloc_and_free(void)
     int size = SIZE_INCREMENTS;
     int loop = ALLOC_LOOP;
     while (loop) {
-        data = malloc(size);
+        data = count < ALLOC_ARRAY_SIZE ? malloc(size) : NULL;
```

Comment:
Also separate PR fixing this issue

Reply:
TESTS/mbedmicro-rtos-mbed/malloc/main.cpp is now submitted as https://github.com/ARMmbed/mbed-os/pull/10977